sdk/storage/azure-storage-file-datalake/tests/recordings/test_file_system.test_list_file_systemss.yaml

```diff
@@ -1,90 +0,0 @@
-interactions:
```

Comment:
Was this removed on purpose? Unused recording due to typo in name?

Reply:
Yeah so this always bothered me that this test had an extra `s`, and it is only present in the sync not async version of this file. Since I was going back and adding another `list_file_systems` test, I wanted to finally fix it (remove additional s, delete old recording, re-record) 😄

Reply:
Ah, I didn't see that edit in the test! Nice, thanks for fixing that up!